hibernate-core/src/main/java/org/hibernate/dialect/H2Dialect.java

```diff
@@ -85,19 +81,19 @@ public H2Dialect() {
 		SequenceInformationExtractor sequenceInformationExtractor = SequenceInformationExtractorH2DatabaseImpl.INSTANCE;
 		try {
 			// HHH-2300
-			final Class h2ConstantsClass = ReflectHelper.classForName( "org.h2.engine.Constants" );
-			final int majorVersion = (Integer) h2ConstantsClass.getDeclaredField( "VERSION_MAJOR" ).get( null );
-			final int minorVersion = (Integer) h2ConstantsClass.getDeclaredField( "VERSION_MINOR" ).get( null );
-			final int buildId = (Integer) h2ConstantsClass.getDeclaredField( "BUILD_ID" ).get( null );
-			if ( buildId < 32 ) {
```

Comment:
Seems your format change doesn't align with Hibernate's convention, which uses extra spaces to improve code readability. Please revert back the format changes (if you are using Intellij IDE, refer to the "Code Style" section in https://hibernate.org/community/contribute/intellij-idea/) so your meaningful semantic changes stand out.